tests/test_cookies.py

```diff
@@ -29,7 +29,7 @@ def dst(self, dt):
 
 
 def utcnow_naive():
```

Comment:
could likely rename this function to utcnow or use the one from misc `from falcon.util.misc import _utcnow` like in test_middleware.py

Reply:
Sounds reasonable, will change.

Reply:
Fixed in latest commit.